app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,47 @@
+class CommentsController < ApplicationController
+
+  def index
+    @comments = Comment.all
+    @post = Post.find(params[:post_id])
+  end
+
+  def new
+      @post = Post.find(params[:post_id])
+      @comment = Comment.new
+  end
+
+  def create
+      @post = Post.find(params[:post_id])
+      @comment = @post.comments.create(comment_params)
```

Comment:
Recommend using bang method (`@post.comments.create!...`) or the conditional (`if @post.comments.create...`)